Updated title_holders to only include modded titles

Charanraja123 opened the pull request:
Removed all titles that belonged in the default game folder

Charanraja123 commented:
I think this clean should be okay for now.

Gr770 commented:
Renamed the files so we aren't overwriting anything. cultures.english.yml and the like can stay as those ARE overwriting the culture names. Test the branches code without 00_flavourization.txt (merge into 00_ctp.txt) and then it's good for upload.

Charanraja123 commented:
I've tested it as much as i could. Only the anglo_saxon localisation was missing. I added it back in.

Also went through the error logs and fixed other small errors. 

Also added a missing norman republic barony localisation as someone pointed out in steam

I think everything is done and Its good for upload
